youtube_dl/extractor/rai.py

```diff
@@ -191,8 +208,12 @@ def _real_extract(self, url):
         timestamp = unified_timestamp(try_get(
             media, lambda x: x['availabilities'][0]['start'], compat_str))
 
-        subtitles = self._extract_subtitles(url, video.get('subtitles'))
-
+        subtitles = {}
+        if '' != video.get('subtitles'):
+            subtitles = self._extract_subtitles(url, video.get('subtitles'))
+        else:
+            if video.get('subtitlesArray'):
```

Comment:
Both URL and array should be tried if available.

Reply:
Done!